### ISSUE TITLE
navbar - Pills with dropdowns - dividers have inconsistent thickness if multiple dividers are present

### ISSUE THREAD

**Issue by iwolford** [twbs/bootstrap#35616]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

When multiple dividers are used in navbar pill dropdowns, first divider is 3px thick, the rest are 2px. Inconsistent styling.

[screenshot](https://imgur.com/k38hFBB)

### Reduced test cases

Code taken right out of code samples, added additional dividers
https://getbootstrap.com/docs/5.0/components/navs-tabs/#pills-with-dropdowns

https://codepen.io/iwolford/pen/XWeVwNr

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v.5.0

**Comment by folknor** [comment]
Hi! Do you mind dragging and dropping the screenshot directly into the github comment text box? imgur is blocked on my network, and even if it wasn't I don't trust them. Thank you!

**Comment by iwolford** [comment]
```<ul class="nav nav-pills">
  <li class="nav-item">
    <a class="nav-link active" aria-current="page" href="#">Active</a>
  </li>
  <li class="nav-item dropdown">
    <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-expanded="false">Dropdown</a>
    <ul class="dropdown-menu">
      <li><a class="dropdown-item" href="#">Action</a></li>
      <li><a class="dropdown-item" href="#">Another action</a></li>
      <li><a class="dropdown-item" href="#">Something else here</a></li>
      <li><hr class="dropdown-divider"></li>
      <li><a class="dropdown-item" href="#">Separated link</a></li>
      <li><hr class="dropdown-divider"></li>
      <li><a class="dropdown-item" href="#">Separated link</a></li>
      <li><hr class="dropdown-divider"></li>
      <li><a class="dropdown-item" href="#">Separated link</a></li> 
    </ul>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#">Link</a>
  </li>
  <li class="nav-item">
    <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
  </li>
</ul>   ```

![divider](https://user-images.githubusercontent.com/12873011/147885257-6d283344-082a-450c-9451-562e72e9f6dc.png)

**Comment by florianlacreuse** [comment]
On my side it only happens when the page is zoomed in. What is your zoom level? 100%?

**Comment by florianlacreuse** [comment]
May be related to #34374.

**Comment by iwolford** [comment]
I confirm my display settings are at 125% Scale (recommended and default setting on my Windows laptop). So it does look like a duplicate of the one you posted above. Thank you!

**Comment by mdo** [comment]
Duplicate of #34374 indeed. Closing for that issue :). Will be shipping a fix in v5.2.0.
